cosmogrb/instruments/gbm/gbm_grb.py

```diff
@@ -74,6 +74,7 @@ def _setup_source(self):
                 0.0,
                 self._duration,
                 source_function,
+                z = self.z,
```

Comment:
we always want to use '_' to keep properties protected from outside modification.

Reply:
Yes, I missed that here